charts/linkerd-crds/templates/policy/authorization-policy.yaml

```diff
@@ -0,0 +1,95 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: authorizationpolicies.policy.linkerd.io
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+  labels:
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    linkerd.io/control-plane-ns: {{.Release.Namespace}}
+spec:
+  group: policy.linkerd.io
+  scope: Namespaced
+  names:
+    kind: AuthorizationPolicy
+    plural: authorizationpolicies
+    singular: authorizationpolicy
+    shortNames: []
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          required: [spec]
+          properties:
+            spec:
+              description: >-
+                Authorizes clients to communicate with Linkerd-proxied server resources.
+              type: object
+              required: [targetRef, requiredAuthenticationRefs]
+              properties:
+                targetRef:
```

Comment:
So both `targetRef` and the items in `requiredAuthenticationRefs` use the same `TargetRef` type in rust code. The spec in the CRD does not define a `namespace` property. Is that intentional? Does that mean that `AuthorizationPolicySpec.target_ref.namespace` will always be `None`. If so, isn't that rather implicit?

Reply:
Yeah, that's true. `TargetRef` is more general (and comes with a variety of helpers). Seemed easier just to reuse the types/helpers than to duplicate them with the minor difference. It's an implementation detail for now, though. Maybe worth separating as things settle.